rego/Drive.rego

```diff
@@ -99,7 +100,8 @@ NonCompliantOUs1_3 contains OU if {
     Events := utils.FilterEvents(LogEvents, "SHARING_OUTSIDE_DOMAIN", OU)
     count(Events) > 0
     LastEvent := utils.GetLastEvent(Events)
-    contains("SHARING_ALLOWED INHERIT_FROM_PARENT", LastEvent.NewValue) == true
+    AcceptableValues := {"SHARING_ALLOWED_WITH_WARNING", "SHARING_NOT_ALLOWED", "INHERIT_FROM_PARENT"}
```

Comment:
`SHARING_NOT_ALLOWED_BUT_MAY_RECEIVE_FILES` needs to be included in the list of acceptable values. Without it, the following configuration results in a failure for 1.3:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/b0e870c4-b9a4-4a93-86bc-00c343feb543)

```suggestion
    AcceptableValues := {"SHARING_ALLOWED_WITH_WARNING", "SHARING_NOT_ALLOWED", "INHERIT_FROM_PARENT",
        "SHARING_NOT_ALLOWED_BUT_MAY_RECEIVE_FILES"}
```

Reply:
The checkbox should be unselected for 1.2 (Allow users to receive files from users or...)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/27ea6790-8d7d-413e-b8c8-dc7a69b6fff9)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/38932956-032b-41b0-8d62-d058564bc5c3)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/fd6d887c-987b-4c79-9f9a-6e100fef3732)

Reply:
The checkbox for 1.2 should be deselected (Allow users to receive files....)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/7efd9859-2c82-4905-a269-bab7f401a322)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/5c2df25e-15aa-4763-8fc1-0b91330e95d2)

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/6ea286bf-116c-49f9-baee-5574d12fe871)

Reply:
Ok, made the updates.
Also, noticed that for 1.1 "SHARING_NOT_ALLOWED_BUT_MAY_RECEIVE_FILES" should be added, so that 1.1 passes even when and if 1.2 fails. This way now all of them 1.1 through 1.4 are independently working. Hope this covers all the use cases and the outputs are as expected.

Reply:
Good catch about 1.1